app/controllers/questions_controller.rb

```diff
@@ -0,0 +1,39 @@
+class QuestionsController < ApplicationController
+  def index
+    @questions = Question.all
+  end
+
+  def new
+    @question = Question.new
+  end
+
+  def create
+    @question = Question.new(params.require(:question).permit(:title, :body, :resolved))
```

Comment:
Looks great @npauzenga. A general Rails convention is to define a private method inside the controller to whitelist your parameters. It's specially useful when you are also permitting the same parameters in the `update` action like you are below. In that case you can just call the method in both instances and if you ever need to allow or disallow parameters, you only need to update them in one place. Something like:

```ruby

...
  @question = Question.new(question_params)
...
  @question.update(question_params)
...


private

def question_params
  params.require(:question).permit(:title, :body, :resolved)
end
```